spec/spec.tex

```diff
@@ -829,6 +829,15 @@ \subsubsection{Read Under Write Behaviour}
 
 If the read-under-write flag is set to \verb|undefined|, then the value held by the read port after the appropriate read latency is undefined.
 
+
+For the purpose of defining such collisions, an ``active write port'' is either a write port or readwrite port that is used to initiate a write operation on a given clock edge.
+An ``active read port'' is a read port or readwrite port that is used to initiate a read operation on a given clock edge.
+Each operation is defined to be ``active'' for the number of cycles set by its corresponding latency, starting from the cycle where its inputs were provided to its associated port.
```

Comment:
I always thought that a read/write was only active in the cycle when it is actually executed, i.e. after `N` (=latency) cycles.

Reply:
Oops, I deleted my comments. The current read-under-write specification implies that a read can be active before that. The definition in the this current spec is meant to be conservative to avoid the other alternative of a huge matrix of behaviors.